packages/xsnap/makefiles/mac/xsnap.mk

```diff
@@ -32,6 +32,7 @@ C_OPTIONS = \
 	-DmxRun=1 \
 	-DmxSloppy=1 \
 	-DmxSnapshot=1 \
+	-DmxMetering=1 \
```

Comment:
https://github.com/Moddable-OpenSource/moddable/issues/550 says the metering code is experimental. I wonder whether we should ask @dtribble or somebody before we merge it to master.

Reply:
@dtribble says ok.